christfellowshipchurch/web-app-v2
# fix: prevent analytics stuff from loading in development

### About

Currently these return statements aren't firing in development, which means some of this tracking code is getting called in development even if you don't intend it to. Cheers!

### Test Instructions

Run in development, notice no `console.warn` about "GoogleAnalytics should be loaded manually before gtag is called.", which is currently logged in master

### Screenshots


### Closes Tickets


### Related Tickets
